components/loading-box/src/stories.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, number, boolean } from '@storybook/addon-knobs/react';
+
+import LoadingBox from '.';
+
+const stories = storiesOf('LoadingBox', module);
+
+stories.addDecorator(withKnobs);
+
+stories.add('LoadingBox default', () => (
+  <LoadingBox
+    loading={boolean('loading', false)}
+    backgroundColor={text('backgroundColor', '#808080')}
+    timeIn={number('timeIn', 200)}
+    timeOut={number('timeOut', 200)}
+    backgroundColorOpacity={number('backgroundColorOpacity', 0.7)}
+    spinnerColor={text('spinnerColor', '#000')}
+  >
+  <h1 style={{ marginTop: 0 }}>Toggle settings under 'knobs'</h1>
+  <p>Ut enim ad minim veniam, quis laboris nisi ut aliquip ex ea commodo consequat.</p>
```

Comment:
Please can you include a few more elements as child components of the example? At least a button and an input field.

Please can you also make sure the content looks GDS-ish, i.e. no times new roman text